integration/helpers/resource.py

```diff
@@ -223,6 +224,7 @@ def _resource_using_s3_events(resource: Dict[str, Any]) -> bool:
         _resource_using_s3_events(resource) for resource in template_dict.get("Resources", {}).values()
     ),
     LOCATION: lambda template_dict, cfn_resource_types: "AWS::Location::PlaceIndex" in cfn_resource_types,
+    APP_SYNC: lambda template_dict, cfn_resource_types: "AWS::AppSync::GraphQLApi" in cfn_resource_types,
```

Comment:
The integ tests failed due to these four resources not available `[AWS::AppSync::GraphQLApi, AWS::AppSync::DataSource, AWS::AppSync::GraphQLSchema, AWS::AppSync::ApiKey]`.
Do we need one for each of them?

Reply:
It seems like not, based on other examples. Also, if `AWS::AppSync::GraphQLApi` is not available, others cannot be available. And if `AWS::AppSync::GraphQLApi` is available, others should be available, since there is AppSync in the region

Reply:
That makes sense.